webui/react/src/e2e/models/components/FilterForm/components/FilterGroup.ts

```diff
@@ -0,0 +1,70 @@
+import { BaseComponent, NamedComponent } from 'e2e/models/BaseComponent';
+import { ConjunctionContainer } from 'e2e/models/components/FilterForm/components/ConjunctionContainer';
+import { FilterField } from 'e2e/models/components/FilterForm/components/FilterField';
+import { DropdownMenu } from 'e2e/models/hew/Dropdown';
+
+/**
+ * Returns a representation of the FilterGroup component.
+ * This constructor represents the contents in src/components/FilterGroup.tsx.
+ * @param {object} obj
+ * @param {CanBeParent} obj.parent - The parent used to locate this FilterGroup
+ * @param {string} obj.selector - Used instead of `defaultSelector`
+ */
+export class FilterGroup extends NamedComponent {
+  readonly defaultSelector = '[data-test-component="FilterGroup"]';
+
+  #childrenSelector = '[data-test="children"]';
+  #notNestedSelector = `:not(${this.#childrenSelector} *)`;
```

Comment:
should these be `private`/`readonly`?

Reply:
yes, let me add readonly. theyre already private